app/jobs/in_person/enrollments_ready_for_status_check/uses_analytics.rb

```diff
@@ -0,0 +1,11 @@
+module InPerson
+  module EnrollmentsReadyForStatusCheck
+    module UsesAnalytics
```

Comment:
Is "uses" an abbreviation, or is it like the very like "this uses that"?

Reply:
`Uses` is a verb here.

Reply:
ok! I feel like that's not a common naming convention we have in this repo? Is there a particular reason to start that now?

Reply:
The use of this convention is pretty isolated, so I don't think it's a big concern. If you want to have the job refactored to use delegation instead of mixins though then I'll rename accordingly.